### ISSUE TITLE
Custom "jumpable" luasnip function in cmp config skips nodes

### ISSUE THREAD

**Issue by bncpr** [LunarVim/LunarVim#2927]
### Problem description

Say you have this Luasnip snippet:
```lua
s("log",
  fmt(
    'LOG({1}, "{2}")',
    {
      c(1, { i(nil, "LOG_ERR"), i(nil, "LOG_DEBUG") }),
      i(2, "err_msg")
    }
  )
)
```
After expansion, you get this:

`LOG(⎵LOG_ERR, "err_msg")`

Where ⎵ is the cursor position. If you want to skip the choice node and press TAB, instead of moving to the next jumpable node i(2), the cursor jumps straight to the exit node at the end of the sentence. I believe this it because of the custom `jumpable` function in `cmp.lua` because when you use the regular `luasnip.jumpable` function, you get the desired behavior.

### LunarVim version

rolling

### Neovim version (>= 0.7.2)

NVIM v0.8.0-dev+580-g45ba2e147

### Operating system/version

ubuntu20.04

### support info

```console
# :LspInfo
 
 Language client log: /home/benc/.local/state/nvim/lsp.log
 Detected filetype:   lua
 
 2 client(s) attached to this buffer: 
 
 Client: sumneko_lua (id: 1, pid: 13739, bufnr: [6])
 	filetypes:       lua
 	autostart:       true
 	root directory:  Running in single file mode.
 	cmd:             lua-language-server
 
 Client: null-ls (id: 2, pid: 5001, bufnr: [4, 5, 6])
 	filetypes:       proto, sh
 	autostart:       false
 	root directory:  /home/benc/dev/LunarVim
 	cmd:             nvim
 
 2 active client(s) not attached to this buffer: 
 
 Client: sumneko_lua (id: 3, pid: 13745, bufnr: [5])
 	filetypes:       lua
 	autostart:       true
 	root directory:  /home/benc/dev/LunarVim
 	cmd:             lua-language-server
 
 Client: sumneko_lua (id: 4, pid: 13761, bufnr: [4])
 	filetypes:       lua
 	autostart:       true
 	root directory:  Running in single file mode.
 	cmd:             lua-language-server
 
 Configured servers list: pylsp, sumneko_lua, clangd
```

```console
# :LvimInfo
                                     
                                     Buffer info
                                     * filetype:                lua
                                     * bufnr:                   6
                                     * treesitter status:       active
                                     
                                     Active client(s)
                                     * name:                      sumneko_lua
                                     * id:                        1
                                     * supported filetype(s):     lua
                                     * attached buffers:          6
                                     * root_dir pattern:          6
                                     * capabilities:              hoverProvider          | typeDefinitionProvider          | documentFormattingProvider
                                                                  documentSymbolProvider | workspaceSymbolProvider         | documentHighlightProvider
                                                                  referencesProvider     | foldingRangeProvider
                                                                  definitionProvider     | documentRangeFormattingProvider
                                     * name:                      sumneko_lua
                                     * id:                        3
                                     * supported filetype(s):     lua
                                     * attached buffers:          5
                                     * root_dir pattern:          5
                                     * capabilities:              hoverProvider          | typeDefinitionProvider          | documentFormattingProvider
                                                                  documentSymbolProvider | workspaceSymbolProvider         | documentHighlightProvider
                                                                  referencesProvider     | foldingRangeProvider
                                                                  definitionProvider     | documentRangeFormattingProvider
                                     * name:                      sumneko_lua
                                     * id:                        4
                                     * supported filetype(s):     lua
                                     * attached buffers:          4
                                     * root_dir pattern:          4
                                     * capabilities:              hoverProvider          | typeDefinitionProvider          | documentFormattingProvider
                                                                  documentSymbolProvider | workspaceSymbolProvider         | documentHighlightProvider
                                                                  referencesProvider     | foldingRangeProvider
                                                                  definitionProvider     | documentRangeFormattingProvider
                                     
                                     
                                     
                                     Formatters info
                                     * Active: 
                                     * Supported: [lua_format, stylua]
                                     
                                     Linters info
                                     * Active: codespell  
                                     * Supported: [luacheck, selene]
                                     
                                     Code actions info
                                     * Active: 
```

**Comment by kylo252** [comment]
please check if the behavior is fixed in #2913

**Comment by bncpr** [comment]
Yes, it seems to be fixed
